server/models/event.js

```diff
@@ -2,8 +2,32 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const eventSchema = new Schema({
-  title: {
-    type: String
+  master_event_uid:   Number,
+  status:             String,
+  start_date_utc:     Date,
+  end_date_utc:       Date,
+  duration:           Number,
```

Comment:
@jberns nice work on this! I'll let Devin offer any feedback he has, my two cents is that this field should be called `duration_in_seconds` instead (or duration_in_minutes) so we know what unit it's using.

Reply:
Do you think we even need duration as its own database entry, I can't remember exactly why it was added in the first place?  We can always calculate this from the start/end time of the event.

Reply:
@jberns You're right, I think that, out of start date, end date, and duration, we only need two of the three. We can either do without end date or without duration.

Reply:
I removed duration as the calendars native input is typically start/end so we don't need to convert between.